.github/workflows/e2e.yml

```diff
@@ -141,9 +141,18 @@ jobs:
           sudo cp modctl/modctl /bin/modctl
           sudo chmod +x /bin/modctl
           modctl version
-          modctl login -u ${{ github.actor }} \
-                       -p ${{ secrets.GITHUB_TOKEN }} \
-                       ${{ env.REGISTRY }}
+
+          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
+            echo "Login using PAT for Pull Request"
+            modctl login -u ${{ github.actor }} \
+                         -p ${{ secrets.CR_PAT }} \
```

Comment:
Can it work for any user who isn't the actual owner of `CR_PAT`?

Reply:
Yes, any user can work well if the type of password is `PAT`.